packages/core/src/dynamoDb/utils/dynamodb.ts

```diff
@@ -242,3 +248,13 @@ export async function deleteItem(
     }
     return await client.deleteItem(deleteRequest)
 }
+
+async function getMaxItemsPerPage(): Promise<number> {
+    return Settings.instance.getSection('aws').get<number>('dynamodb.maxItemsPerPage', 100)
+}
+
+export function dynamoDbConsoleUrl(node: DynamoDbTableNode): vscode.Uri {
+    const service = 'dynamodb'
+    const resourcePath = `tables:selected=${node.dynamoDbtable}`
+    return vscode.Uri.parse(`https://console.aws.amazon.com/${service}/home?region=${node.regionCode}#/${resourcePath}`)
```

Comment:
I'm not sure if this will work in regions like china btw. I could be wrong but I don't think aws exists there